### PR TITLE
Fixed Responsive issue in Project/Meshery 

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil Sharma <nikhilsharmamusic2000@gmail.com>

**Description**
- There is a responsive issue in the project/meshery page for smaller screens.

** Screenshot **
Screenshot from 2021-08-06 14-13-27

## After Changes
- aligned the content in the middle

** Screenshot **
Screenshot from 2021-08-06 14-09-33

**Notes for Reviewers**
Let me know if there are any other changes we can make.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5bb7402b0ace1947682c259506e1ca61cd7f5813 at: https://610cfccb45f65d86563f2346--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
Oooh, This looks great on my phone, good work @NikhilSharma03

File: src/sections/Meshery/meshery.style.js [hunk 1]
```diff
@@ -100,13 +100,16 @@ const MesheryWrapper = styled.section`
...
             .description {
                 padding: 0;
+                flex-direction: column-reverse;
                 .meshery-hero-img {
                     text-align: center;
                     padding: 0;;
                 }
                 .desc-text{
+                    text-align: center;
+                    margin-bottom: .75rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/meshery.style.js:110]: make the `max-width: 100%` as well for this section.

File: src/sections/Meshery/meshery.style.js [hunk 2]
```diff
@@ -115,6 +118,15 @@ const MesheryWrapper = styled.section`
...
     }
+    @media only screen and (max-width: 479px) {
+        .info{
+            .description {
+                .desc-text{
+                    max-width:90%
+                }
+            }
+        }
+    }
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/meshery.style.js:129]: After the above change, we don't need this.

Comment (NikhilSharma03) [reply src/sections/Meshery/meshery.style.js:129]: OK, I will make these changes.

**Comment by NikhilSharma03** [comment]
Sorry, I think I have accidentally added the package files also in the previous command.
@Nikhil-Ladha Can you please check now?

**Comment by l5io** [comment]
🚀 Preview for commit 7074c28cde027705611c70d31f7c4e1d52a1d72a at: https://610d19568e155cac6afb14cd--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**